README.md

```diff
@@ -176,6 +176,8 @@ Create a verifier function by calling `createVerifier` and providing one or more
 
 - `clockTolerance`: Timespan in milliseconds is the tolerance to apply to the current timestamp when performing time comparisons. Default is `0`.
 
+- `tokenHasher`: The function that will be used to hash the token. By default is used [the hashToken function](./src/utils.js).
```

Comment:
I would add some details, perhaps in the dedicated caching section of the readme, about this option, and what it is used for. Also, I would name it more explicitly. I assume that the only reason we hash the token is for the cache key, right? If so, the name and docs above should reflect that, otherwise a user would not know what this is for. Basically, let's be a bit more comprehensive with the docs